geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitor.java

```diff
@@ -582,7 +581,7 @@ boolean doTCPCheckMember(ID suspectMember, int port,
       } catch (IOException e) {
         // this is expected if it is a connection-timeout or other failure
         // to connect
-      } catch (IllegalStateException | GemFireConfigException e) {
+      } catch (IllegalStateException e) {
```

Comment:
Should we remove this whole catch block? If it was good to propage GemFireConfigException up, maybe we should do that for IllegalStateException as well?

Reply:
we could do that... only action we took was to potentially log a small message...

Reply:
https://issues.apache.org/jira/browse/GEODE-7696